index.html

```diff
@@ -11742,6 +11743,7 @@ <h2>Index</h2>
    <!-- Simple dialogs --> <li><dfn data-lt="simple dialog"><a href=https://html.spec.whatwg.org/#simple-dialogs>Simple dialogs</a></dfn>
    <!-- Steps to fire beforeunload --> <li><dfn><a href=https://html.spec.whatwg.org/#steps-to-fire-beforeunload>Steps to fire beforeunload</a></dfn>
    <!-- Suffering from bad input --> <li><dfn><a href=https://html.spec.whatwg.org/#suffering-from-bad-input>Suffering from bad input</a></dfn>
+   <!-- Top-level browsing context of --> <li><dfn><a href=https://html.spec.whatwg.org/#bc-tlbc>Top-level browsing context of</a></dfn>
```

Comment:
I think we should probably leave of out of the definition and only have "top-level browsing context" as the definition

Reply:
https://w3c.github.io/webdriver/#dfn-browsing-contexts:~:text=to%20capture%20the-,top%2Dlevel%20browsing%20context,-%27s%20initial%20viewport

Reply:
I believe scoped concepts are applicable here https://github.com/speced/respec/wiki/Shorthands-Guide#scoped-concepts (example in https://github.com/w3c/webdriver/pull/1849/commits/bc1b1cfc409c527030f68ffcf49b9a26368e2746)

Reply:
applied